packages/apollo-server-lambda/src/lambdaApollo.ts

```diff
@@ -32,9 +32,12 @@ export function graphqlLambda(
     callback,
   ): void => {
     context.callbackWaitsForEmptyEventLoop = false;
-    let { body, isBase64Encoded } = event;
+    let { body, headers, isBase64Encoded } = event;
+    let query: Record<string, any> | Record<string, any>[];
+    const contentType = headers["content-type"] || headers["Content-Type"];
+    const isMultipart = (contentType || "").startsWith("multipart/form-data");
```

Comment:
I would suggest to lowercase `contentType` just to be sure

Reply:
Done, thanks for the review.
Do you know how to help this PR to be merged faster?

Reply:
sorry :(